website/static/js/oauthAddonNodeConfig.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Module that controls the addon node settings. Includes Knockout view-model
+ * for syncing data, and HGrid-folderpicker for selecting a folder.
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout.punches');
```

Comment:
What do you mean by "not necessarily available"? Basic knockout can cover the two uses I see in the template; this particular library usage just seems gratuitous.